### PR TITLE
🏗 PR-check logging cleanup

### PR DESCRIPTION
**Pull request by rsimha**
This is part 2 of a logging cleanup in anticipation of the move from Travis to CircleCI. Part 1 was in #32041. There are no major changes to how the logs look during CI, but the logging code has been simplified a fair bit.

**PR highlights:**

- Reusable logging prefixes (no more passing around `FILENAME` args)
- Helper `printSkipMessage()` for when jobs are skipped
- Abbreviate log-coloring by destructuring `require('ansi-colors')`
- Clean up presubmit allowlists for `build-system/`

**Coming up:** Remaining logic fixes and function renames.

### CODE REVIEW

**Review by danielrozenberg — COMMENTED** [review]
It's really difficult to review this since there's a mix of refactors, renames, and logic changes... Any chance you can split this into multiple, smaller PRs?

**Comment by rsimha** [comment]
> It's really difficult to review this since there's a mix of refactors, renames, and logic changes... Any chance you can split this into multiple, smaller PRs?

Fair point. I've reduced this PR to contain only the logging changes. Will do function renames and logic fixes in a follow up PR.

File: build-system/pr-check/unminified-tests.js [hunk 1]
```diff
@@ -51,7 +50,7 @@ function main() {
         'Integration tests failed! Skipping remaining tests.'
       );
       timedExecOrThrow(
-        'gulp unit --nobuild --headless --coverage --report',
+        'gulp unit --headless --coverage --report',
```
Comment (danielrozenberg) [inline build-system/pr-check/unminified-tests.js:53]: Why this change?

**Review by danielrozenberg — APPROVED** [review]

File: build-system/pr-check/unminified-tests.js (hunk 1 above)
Comment (rsimha) [reply build-system/pr-check/unminified-tests.js:53]: Oh, I meant to do this in the next PR (coming up shortly). Since `--nobuild` is effectively a no-op for unit tests, so I'm going to leave this in for now.
